samples/wifi/shell/src/wifi_cred_shell.c

```diff
@@ -0,0 +1,229 @@
+/*
```

Comment:
Perhaps we should just extend `wifi_shell.c` with new commands if `wifi_mgmt_ext` is enabled?

Reply:
We can later move it there if we decide to upstream the autoconnect feature.